internal/controllers/cassandra/cassandradatacenter_controller.go

```diff
@@ -117,7 +122,7 @@ func (r *CassandraDatacenterReconciler) Reconcile(ctx context.Context, request c
 
 		// Error reading the object
 		logger.Error(err, "Failed to get CassandraDatacenter.")
-		return ctrl.Result{RequeueAfter: 10 * time.Second}, err
+		return ctrl.Result{}, err
```

Comment:
Kinda, but not really. It's because returning non-nil err will cause the requeue to happen in controller-runtime, so we should never set Requeue time in that case.